### ISSUE TITLE
deepcell.model_zoo.fpn.__create_semantic_head applies ReLU for n_channels=1

### ISSUE THREAD

**Issue by JLrumberger** [vanvalenlab/deepcell-tf#624]
**Describe the bug**

`deepcell.model_zoo.fpn.__create_semantic_head` which is used by default in your segmentation models applies ReLU activation function on the output if `n_classes == 1`. I would expect that segmentation models with single channel output use sigmoid activation in their last layer. Furthermore, applying ReLU makes the logits unusable for `binary_cross_entropy_with_logits` since sigmoid(relu(x)) has a lower bound of 0.5.

**To Reproduce**

Something like the below code would return values bound between [0, inf]

```
x = tf.constant(np.random.rand(1,256,256,2), tf.float32)
model = PanopticNet(
    backbone="resnet50", input_shape=[256,256,2],
    norm_method="std", num_semantic_classes=[1],
)
out = model(x)
print(tf.reduce_min(tf.sigmoid(out)))
```

**Expected behavior**
Expected would be that the output is run through sigmoid, not ReLU.

**Desktop (please complete the following information):**
 - OS: Win 10
 - deepcell 0.12.3
 - Python 3.8.0
